templates/archive.jade

```diff
@@ -0,0 +1,34 @@
+
+extends layout
+//- this logic should be moved to a view at some point
+
+block content
+	- var lineHeight = 2.2;
+	- var archives = _.chain(env.helpers.getArticles(contents)).groupBy(function(item) {
+	-	 return item.date.getFullYear()
+	- }).value()
+	- for (var archive in archives) {
+	-	archives[archive] = _.groupBy(archives[archive], function(item){return item.date.getMonth();})
+	- }
+	- var month_names = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
+	section.archive
+		h2 Archive
+		ul
+			- var yearsK = _.chain(archives).keys().reverse().value()
+			- for(var year in yearsK)
+				- months = archives[yearsK[year]]
+				- var yearHeight = lineHeight * _.reduce(months, function(memo,month) { return memo + month.length; }, 0);
+				li
+					span.year-label(style='line-height:' + yearHeight+'em')=yearsK[year]
```

Comment:
Use double quotes for Jade. Spacing inconsistent for `+'s`. Need space between `=` and `yearsK`